bruno/SIS/v4/EducationOrganization/CalendarDates/01 - Check CalendarDate is valid.bru

```diff
@@ -0,0 +1,75 @@
+meta {
+  name: 01 - Check CalendarDate is valid
+  type: http
+  seq: 1
+}
+
+get {
+  url: {{resourceBaseUrl}}/ed-fi/CalendarDates?schoolId={{tempSchoolId}}&calendarCode={{tempCalendarCode}}
+  body: none
+  auth: bearer
+}
+
+params:query {
+  schoolId: {{tempSchoolId}}
+  calendarCode: {{tempCalendarCode}}
+  ~id: {{tempCalendarDateUniqueId}}
+  ~date: 
+  ~schoolYear: 
+}
+
+auth:bearer {
+  token: {{edFiCertToken}}
+}
+
+assert {
+  res.status: eq 200
+  res.body: isArray
+  res.body[0].id: isDefined
+  res.body[0].date: isString
+  res.body[0].calendarReference: isDefined
+  res.body[0].calendarReference.calendarCode: isString
```

Comment:
`calendarCode` is listed as an `int` in the documentation below, though I suspect that is wrong.

Should this line test to ensure that the `calendarCode !== ""`? Does the `isString` function already cover that, or would that be through use of `isNotEmpty`?